packages/nvidia_nat_memmachine/tests/test_memmachine_editor.py

```
@@ -0,0 +1,510 @@
+    assert "tags" in user_call_data["metadata"]
+    
+    # Verify assistant message
+    assistant_call_data = next(c for c in calls_data if c["role"] == "assistant")
+    assert assistant_call_data["content"] == "Hello Alex! I've noted that you're a vegetarian and have a nut allergy."
+    # Now uses memory_types instead of episode_type
+    assert assistant_call_data["episode_type"] is None
+
+
+async def test_add_items_with_direct_memory(
+    memmachine_editor_with_client: MemMachineEditor,
+    mock_client: Mock,
```

Ruff (0.14.14)

Unused function argument: `mock_client`

(ARG001)